app/forms/otp_delivery_selection_form.rb

```diff
@@ -15,7 +15,7 @@ def initialize(user, phone_to_deliver_to, context)
 
   def submit(params)
     self.otp_delivery_preference = params[:otp_delivery_preference]
-    self.resend = params[:resend]
+    self.resend = params[:resend] == true || params[:resend] == 'true'
```

Comment:
Would `params` values be anything other than strings? (i.e. do we need the first part of the condition?)

```suggestion
    self.resend = params[:resend] == 'true'
```